code/index.html

```diff
@@ -2,21 +2,141 @@
+
+        <div class="small-news-container">
+          <img src="images/waterfall.jpg" alt="Waterfall">
+          <span class="credit-text">Photo by <a class="credit-text-links" href="https://unsplash.com/@robertlukeman?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Robert Lukeman</a> on <a class="credit-text-links" href="https://unsplash.com/s/photos/waterfall?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Unsplash</a></span>
+          <h4>Seljalandsfoss Waterfall, Iceland</h4>
+          <a class="read-more-links" href="#"> Click here to read more..</a>
+        </div>
+
+        <div class="small-news-container">
+          <img src="images/deutschland.jpg" alt="Watzmanngletscher">
+          <span class="credit-text">Photo by <a class="credit-text-links" href="https://unsplash.com/@danielsessler?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Daniel Seßler</a> on <a class="credit-text-links" href="https://unsplash.com/t/wallpapers?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Unsplash</a></span>
+          <h4>Watzmanngletscher, Deutschland</h4>
+          <a class="read-more-links" href="#"> Click here to read more..</a>
+        </div>
+
+        <div class="small-news-container">
+          <img src="images/orange-cat.jpg" alt="Orange cat">
+          <span class="credit-text">Photo by <a class="credit-text-links" href="https://unsplash.com/@mannydream?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Manny Moreno</a> on <a class="credit-text-links" href="https://unsplash.com/s/photos/reykjavik?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Unsplash</a></span>
+          <h4>Orange cat, Iceland</h4>
+          <a class="read-more-links" href="#"> Click here to read more..</a>
+        </div>
+      </section>
+
+      <!-- FOOTER -->
+      <footer>
+        <ul class="first-footer-section">
+          <li><a href="#">About us</a></li>
+          <li><a href="#">What we do</a></li>
+          <li><a href="#">Our team</a></li>
+          <li><a href="#">Contact us</a></li>
+        </ul>
+        <ul class="second-footer-section">
+          <li><a href="#">Integrety policy</a></li>
+          <li><a href="#">Cookie policy</a></li>
+          <li><a href="#">Terms of use</a></li>
+          <li><a href="#">Invoicing info</a></li>
+        </ul>
+        <ul class="third-footer-section">
+          <li><img src="https://img.icons8.com/ios-filled/50/000000/facebook-f.png"/><a href="#">Facebook</a></li>
+          <li><img src="https://img.icons8.com/ios-filled/50/000000/instagram-new.png"/><a href="#">Instagram</a></li>
+          <li><img src="https://img.icons8.com/ios-filled/50/000000/youtube.png"/><a href="#">YouTube</a></li>
+          <li><img src="https://img.icons8.com/ios-filled/50/000000/twitter.png"/><a href="#">Twitter</a></li>
+        </ul>
+      </footer>
```

Comment:
If you wanna practice CSS Grid, you could give this footer a try! In mobile, it looks a little cramped with three columns 😇 
<img width="367" alt="Skärmavbild 2021-02-03 kl  13 24 45" src="https://user-images.githubusercontent.com/51529201/106746711-3096a680-6623-11eb-9879-e3e6e74fffa5.png">